src/xdsl/dialects/std.py

```diff
@@ -41,3 +43,25 @@ class Return(Operation):
     @staticmethod
     def get(*ops: Union[Operation, SSAValue]) -> Return:
         return Return.build(operands=[[op for op in ops]])
+
+
+@irdl_op_definition
+class Branch(Operation):
+    name: str = "std.br"
+    arguments = VarOperandDef(AnyAttr())
+
+    @staticmethod
+    def get(*ops: Union[Operation, SSAValue], successor=Block) -> Branch:
```

Comment:
Does this works as expected? Maybe we should move successor before ops?

Reply:
I added some tests and it should now work as expected.